### ISSUE TITLE
Feature request: Localized messages in the view

### ISSUE THREAD

**Issue by greystate** [ryanlewis/seo-metadata#7]
This package does a very good job of helping the editors get a grasp on what's usually just "the _SEO_ stuff" to them. We'd really like to provide the info in the editors' native language though, so they have a coherent, awesome feel about using Umbraco.

For now, we've just manually translated the view page, but that's kind of messy, and will probably break in all kinds of ways whenever there's an update to the package. So have you considered using Dictionary items or similar to enable this in the future? (I'm actually not entirely sure what the official "way" of localizing data types is...)

**Comment by ryanlewis** [comment]
It's something I definitely see being useful. Unfortunately I don't have much experience with the dictionary or multilingual sites and it isn't a feature our clients will benefit, but will investigate and have a look to see what's involved in this. 

Of course, would always welcome a pull request! :wink:

**Comment by greystate** [comment]
Cool - thanks; I'll have a look at it then :)

**Comment by ryanlewis** [comment]
Think this will wait until 7.3 at the earliest, due to some changes in the way localisation is done in that version. Check the [roadmap](https://our.umbraco.org/contribute/roadmap)

**Comment by greystate** [comment]
Agreed - I was investigating this, and found that although there is a `<localize>` directive, it's of limited use because we need to modify the global localization files, or do some hacking to provide our own keys.

**Comment by greystate** [comment]
For myself and others: This did indeed make it into Umbraco v7.3.0 and has since been documented: https://our.umbraco.org/documentation/Extending/Language-Files/Language-Files-For-Packages/

Will be doing this to our local version and throw a PR this way when it's in a reasonably working state.

**Comment by ryanlewis** [comment]
Brilliant. Look forward to that.

**Comment by ryanlewis** [comment]
Closing — this repository is archived. The package targets Umbraco 7, which has been unsupported upstream for years.
